src/app/code/community/Shopgate/Cloudapi/Model/SalesRule/Condition.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+class Shopgate_Cloudapi_Model_SalesRule_Condition extends Mage_Rule_Model_Condition_Abstract
+{
+    /**
+     * Define identifier for cart type
+     */
+    const CART_TYPE = 'shopgate_cart_type';
+
+    /**
+     * Define identifier for select
+     */
+    const DEFAULT_IDENTIFIER_SELECT = 'select';
+
+    /**
+     * @return $this
+     */
+    public function loadAttributeOptions()
+    {
+        $this->setData('attribute_option',
+            array(
+                self::CART_TYPE => Mage::helper('shopgate_cloudapi')->__('Shopgate Cart Type')
```

Comment:
guess what, translations will need to be added to German on all these :D ha!

Reply:
I added them except for the value "Shopgate Mobile App" is fine in german too.